pipeline/action_watcher.go

```diff
@@ -110,7 +110,11 @@ func (aw *actionWatcher) setEventBefore(index int, event *Event) {
 		if s == nil || len(s.eventBefore) > 0 {
 			continue
 		}
-		s.eventBefore = event.Root.EncodeToByte()
+
+		s.eventBefore = []byte("<timeout event>")
```

Comment:
What do you think about creation this slice only once? For example, we can create global var here